packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/testMessageAuthors.st

```diff
@@ -0,0 +1,17 @@
+testing
+testMessageAuthors
+
+	| message user chat_id notification |
+	self receivedMessage: nil.
+	chat_id := ((self core chats) first) id.
+	message := self mockMessageFrom: chat_id.
+
+	self core handleEvent: message.
+	user := self mockUserWith: 1673511834 and: ('9447669760' , chat_id).
+	self core handleEvent: user.
+	
+	notification := TCUNotification newWithMessage: self receivedMessage.
+	
+	self assert: ('Test User' = (notification sender fullName)).
+	self assert: (chat_id = (notification chat id)).
+	self assert: ('Hallo' = (notification message message))
```

Comment:
Given that you are comparing those values to what you set in the other messages (mock user / message), I think having them as class-side constants would be much cleaner

Reply:
Or alternatively, you could directly look them up from the mock message / mock user you have here, but constants seem nicer to me :)